utils/dyn-config.c

```diff
@@ -167,11 +167,20 @@ glusterBlockGetOption(const char *key)
         } while (0);
 
 static void
-glusterBlockConfSetOptions(gbConfig *cfg, bool reloading)
+glusterBlockConfSetOptions(gbConfig *cfg, bool getLogDir)
```

Comment:
May be you want to change the same in the declaration too, in common.h ?
```

- utils/common.h:int glusterBlockLoadConfig(gbConfig *cfg, bool reloading);
+ utils/common.h:int glusterBlockLoadConfig(gbConfig *cfg, bool getLogDir);
```

Reply:
Will update it.
Thanks.